source/includes/documentation/_vue_js_elements.md

```diff
@@ -368,18 +379,46 @@ explicitly through a `JDIBreadcrumbs` annotation:
     }
 ```
 
```

Comment:
Please, update this test as it was refactored

Reply:
Done